### PR TITLE
removed Battery on startup, added Translation; bugfix in Garage output

### PR DESCRIPTION
**Pull request by fel115**
Issue #104 should be fixed with this PR

### CODE REVIEW

File: lang/de_AT.po [hunk 1]
```diff
@@ -19,232 +19,194 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
 "X-Generator: Gtranslator 3.38.0\n"
 
-#: src/main.rs:65
-msgid "failed to execute process"
-msgstr "Ausführen des Prozesses fehlgeschlagen"
+#: src/audio.rs:75
+msgid "🔔 Ringing the Audio Bell "
```
Comment (markus2330) [inline lang/de_AT.po:23]: Needs fix in src

```suggestion
msgid "🔔 Ringing the Audio Bell"
```

File: lang/de_AT.po [hunk 2]
```diff
@@ -19,232 +19,194 @@ msgstr ""
...
+msgstr "🔔 Audio Glocke läutet "
 
-#: src/main.rs:75
+#: src/audio.rs:88
+msgid "🚨 Audio Fire Alarm!"
+msgstr "🚨 Audio Feueralarm"
```
Comment (markus2330) [inline lang/de_AT.po:28]:
```suggestion
msgstr "🚨 Audio Feueralarm!"
```

File: lang/de_AT.po [hunk 3]
```diff
@@ -19,232 +19,194 @@ msgstr ""
...
+#: src/buttons.rs:536
+msgid "{}. {}"
+msgstr "{}. {}"
 
-#: src/main.rs:105
-msgid "💨 Airquality is ok. {}"
-msgstr "💨 Luftqualität ist ok. {}"
+#: src/buttons.rs:556
+msgid "🔔 Pressed button bell."
+msgstr "🔔 Knopf für Glocke gedrückt. {}"
```
Comment (markus2330) [inline lang/de_AT.po:56]: Should have same arguments, shouldn't it?

```suggestion
msgstr "🔔 Knopf für Glocke gedrückt."
```

File: lang/de_AT.po [hunk 4]
```diff
@@ -19,232 +19,194 @@ msgstr ""
...
+#: src/buttons.rs:565
+msgid "🔕 Did not ring bell (button was pressed) because the time 🌜 is {}, {}"
+msgstr "🔕 Läute Glocke nicht (Knopf gedrückt) weil Uhrzeit 🌜 ist {}, {}"
 
-#: src/main.rs:112
-msgid "💩 Airquality is bad! {}"
-msgstr "💩 Luftqualität ist schlecht! {}"
+#: src/buttons.rs:577 
+msgid "💡 Pressed switch inside. {}."
+msgstr "💡 Taster innen gedrückt. {}. {}"
```
Comment (markus2330) [inline lang/de_AT.po:64]: Again different number of arguments?

File: lang/de_AT.po [hunk 5]
```diff
@@ -19,232 +19,194 @@ msgstr ""
...
+#: src/buttons.rs:588
+msgid "💡 Pressed switch outside or light button. {}."
+msgstr "💡 Taster oder Knopf außen gedrückt. {}. {}"
 
-#: src/main.rs:138
-msgid "🚨 Possible fire alarm! (don't ring yet). {}"
-msgstr "🚨 Möglicher Feueralarm! (läute noch nicht). {}"
+#: src/buttons.rs:601
+msgid "🔔 Pressed switch bell."
+msgstr "🔔 Taster Glocke gedrückt. {}"
```
Comment (markus2330) [inline lang/de_AT.po:72]: again

File: src/signals.rs [hunk 6]
```diff
@@ -100,16 +100,6 @@ impl<'a> Signals<'a> {
 				.send(EnvEvent::RestoreBaseline)
 				.await?;
 		}
-		self.nextcloud_sender
```
Comment (markus2330) [inline src/signals.rs:103]: Why remove this completely? There should be a message when reloading config&state via signals.

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thank you, many improvements :heart: 
I tried to resolve the conflict but CI does not build anymore.

**Comment by markus2330** [comment]
please also pump version

File: src/signals.rs (hunk 6 above)
Comment (fel115) [reply src/signals.rs:103]: Sorry thought it was the same as at the beginning of the function.

**Comment by markus2330** [comment]
Great job, thank you, finally finished :1st_place_medal:

**Pull request merged**